src/debug/Debug.js

```diff
@@ -0,0 +1,112 @@
+import dat from 'dat.gui/build/dat.gui';
```

Comment:
Yeah, and I tried to make babel have a pass on it (which is already a hack), but without success :-/